cmd/kaspawallet/balance.go

```diff
@@ -0,0 +1,53 @@
+package main
+
+import (
+	"fmt"
+	"github.com/kaspanet/kaspad/cmd/kaspawallet/keys"
+	"github.com/kaspanet/kaspad/cmd/kaspawallet/libkaspawallet"
+	"github.com/kaspanet/kaspad/infrastructure/network/rpcclient"
+
+	"github.com/kaspanet/kaspad/util"
+)
+
+func balance(conf *balanceConfig) error {
+	client, err := rpcclient.NewRPCClient(conf.RPCServer)
```

Comment:
You should add this everywhere you use the RPCServer, this allows not passing `--rpcserver` to the CLI
```go
	rpcAddress, err := conf.NetParams().NormalizeRPCServerAddress(conf.RPCServer)
	if err != nil {
		return err
	}
```